test/units/testsuite-19.sh

```diff
@@ -36,6 +36,35 @@ systemd-run --wait --unit=test4.service true
 # And now check again, "io" should have vanished
 grep -qv io /sys/fs/cgroup/system.slice/cgroup.controllers
 
+
+# And now for something (not) completely different...
+if [ ! -f /sys/fs/cgroup/init.scope/cgroup.type ] ; then
+	echo "Skippint TEST-19-DELEGATE threads test, cgroup v2 doesn't support cgroup.type" >&2
+	exit 0
+fi
+
+SERVICE_PATH="$(mktemp /etc/systemd/system/test-delegate-XXX.service)"
+SERVICE_NAME="${SERVICE_PATH##*/}"
+
+cat >"$SERVICE_PATH" <<EOF
+[Service]
+Delegate=true
+ExecStartPre=/bin/mkdir /sys/fs/cgroup/system.slice/$SERVICE_NAME/subtree
+ExecStartPre=/bin/bash -c "echo threaded >/sys/fs/cgroup/system.slice/$SERVICE_NAME/subtree/cgroup.type"
+ExecStart=/bin/sleep 86400
+ExecReload=/bin/echo pretending to reload
```

Comment:
Please drop the `/bin/` prefixes, they just make the whole thing harder to read.

Also `sh` instead of `bash` to save some cycles.